ddtrace/span.py

```diff
@@ -127,6 +128,11 @@ def __init__(
         if not (parent_id is None or isinstance(parent_id, six.integer_types)):
             raise TypeError("parent_id must be an integer")
 
+        self._tracer = None
```

Comment:
should we also update the docstring for the `tracer` argument?

Reply:
I updated the doc string to `tracer property is deprecated, this field should be set to None`